isvcs/zk.go

```diff
@@ -61,82 +74,97 @@ func initZK() {
 		Volumes: map[string]string{"data": "/var/zookeeper"},
 	}
 
-	defaultHealthCheck := healthCheckDefinition{
-		healthCheck: zkHealthCheck,
-		Interval:    DEFAULT_HEALTHCHECK_INTERVAL,
-		Timeout:     DEFAULT_HEALTHCHECK_TIMEOUT,
-	}
+	// setup the health check definitions
+	Zookeeper.HealthChecks 	= make(map[string]healthCheckDefinition)
 
-	Zookeeper.HealthChecks = map[string]healthCheckDefinition{
-		DEFAULT_HEALTHCHECK_NAME: defaultHealthCheck,
+	for _, zkIP := range zkConnectStrings {
```

Comment:
This loop is wrong. In cases where zkConnectStrings has 3 or more stanzas, the value used for the isvc is going to be the _last_ entry, but it should be the entry for the localhost.   Thinking a little more about this, I don't believe that the we should make assumptions about the order of IPs in the `SERVICED_ZK` configuration string. For the purposes of running a healthcheck against the *local* ZK instance, we should always use `127.0.0.1:2181` because that's the only value we can count on to find the local ZK instance.